migrating/checklists/ossm-migrating-premigration-checklists-assembly.adoc

```diff
@@ -6,5 +6,121 @@ include::_attributes/common-attributes.adoc[]
+
+* [ ] Disable tracing in your `ServiceMeshControlPlane` resource: `spec.tracing.type=None`
+** [ ] Configure the `ServiceMeshControlPlane` with OpenShift Distributed Tracing as the replacement: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-configuring-distr-tracing-tempo_observability[Configuring {TempoName} and the Red{nbsp}Hat build of OpenTelemetry].
+
+* [ ] Disable Kiali in your `ServiceMeshControlPlane` resource: `spec.addons.kiali.enabled=false`
+** [ ] If you did not create a standalone `Kiali` resource as part of Prometheus or tracing, see: "Using {KialiProduct}".
+
+[id="migrate-to-explicitly-managed-routes_{context}"]
+== Migrate to explicitly managed routes
+
+Automatic route creation, also known as Istio OpenShift Routing (IOR), is a deprecated feature that is disabled by default for any `ServiceMeshControlPlane` resource created using {SMProduct} 2.5 and later. To move from {SMProduct} 2 to {SMProduct} 3, you need to migrate from IOR to explicitly-managed routes.
+
+If you already moved to explicitly-managed routes in {SMProduct} 2, then continue to gateway injection.
+
+* [ ] Migrate from Istio OpenShift Routing (IOR) to to explicitly-managed routes: link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-route-migration[Service Mesh route migration].
+
+[id="migrate-to-gateway-injection_{context}"]
+== Migrate to gateway injection
+
+Gateways were controlled by the `ServiceMeshControlPlane` (SMCP) resource in {SMProduct} 2. The {SMProduct} 3 control plane does not manage gateways so you must migrate from SMCP-Defined gateways to gateway injection.
+
+* [ ] link:https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-gateway-migration[Service Mesh gateway migration].
+
+[id="disable-network-policy-management_{context}"]
+== Disable network policy management
+
+If you do not want your network policies in place during your migration:
+
+* [ ] Disable network policy management in the {SMProduct} 2 `ServiceMeshControlPlane` resource: `spec.security.manageNetworkPolicy=false`.
+* [ ] Complete the rest of the checklists.
+* [ ] Migrate your deployment and workloads.
+* [ ] Manually recreate your network policies after you have migrated your workloads.
+
+If you want your network policies in place during your migration:
+
+* [ ] Manually xref:../../migrating/checklists/ossm-migrating-network-policies-assembly.adoc#ossm-migrating-network-policies-setup-during-migration_ossm-migrating-network-policies-assembly[set up network policies to use during migration].
+* [ ] Disable network policy management in the {SMProduct} 2 `ServiceMeshControlPlane` resource: `spec.security.manageNetworkPolicy=false`.
+* [ ] Complete the rest of the checklists.
+* [ ] Migrate your deployment and workloads.
```

Comment:
This section reads like a procedure to me, so I would suggest putting it in a procedure module and maybe a background of why disabling network policy management is a necessary consideration. As it is, this heading doesn't have an introduction.

Reply:
You must have some vociferous users 😆 but OK, heard and potential for this to be a followup item in a jira. Its not a blocker.